hub/powertoys/run.md

```diff
@@ -14,7 +14,7 @@ PowerToys Run is a quick launcher for power users that contains some additional
 To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _(Note that this shortcut can be changed in the settings window.)_
 
 > [!IMPORTANT]
-> After installing, PowerToys must be enabled and running in the background for this utility to work.
+> PowerToys must be enabled and running in the background for this utility to work.
```

Comment:
PowerToys needs to be enabled?  that is awkward.  PowerToys must be running as well as PT Run be enabled?